src/components/userGroups/UserGroup.tsx

```diff
@@ -639,30 +647,32 @@ const CollapsedUserOrGroup: React.FC<CollapsedUserOrGroupProps> = ({
                 </span>
                 <span
                     className="user-list__direction-container flex rotate pointer"
-                    onClick={email_id === 'admin' ? noop : (e) => setCollapsed(not)}
+                    onClick={onClickHandler}
                     style={{ ['--rotateBy' as any]: collapsed ? '0deg' : '180deg' }}
                 >
                     <ConditionalWrap
-                        condition={email_id === 'admin'}
+                        condition={decision}
                         wrap={(children) => (
-                            <Tippy
+                            <Tippy 
                                 className="default-tt"
                                 arrow={false}
                                 placement="top"
-                                content="Admin user cannot be edited"
+                                content={`${email_id === 'admin' ? 'Admin' : 'System'} user cannot be edited`}
                             >
                                 <div className="flex">{children}</div>
                             </Tippy>
                         )}
+                        
                     >
-                        {email_id === 'admin' ? (
+                        {decision? (
```

Comment:
It will be true in case email is `system` but here we need to check for only `àdmin`

Reply:
Here in this task I am supposed to make  the system user permission also as non-editable ,so I have added system in this condition.